README.md

```diff
@@ -4,52 +4,7 @@ This repository is used to store styleguides for Mozilla communities. If you’r
 
 ## Style Guides
 
-* [Mozilla general style guide](mozilla_general/README.md)
-* [Africaans (af)](af/README.md)
-* [Arabic (ar)](ar/README.md)
-* [Armenian (hy-AM)](hy-AM/README.md)
-* [Bambara (bm)](bm/README.md)
-* [Bengali, Bangladesh (bn-BD)](bn-BD/README.md)
-* [Chinese, Simplified (zh-CN)](zh-CN/README.md)
-* [Chinese, Traditional (zh-TW)](zh-TW/README.md)
-* [Czech (cs)](cs/README.md)
-* [Dutch (nl)](nl/README.md)
-* [Esperanto (eo)](eo/README.md)
-* [Finnish (fi)](fi/README.md)
-* [German (de)](de/README.md)
-* [Hausa (ha)](ha/README.md)
-* [Hindi (hi-IN)](hi-IN/README.md)
-* [Icelandic (is)](is/README.md)
-* [Igbo (ig)](ig/README.md)
-* [Irish (ga-IE)](ga-IE/README.md)
-* [Italian (it)](it/README.md)
-* [Japanese (ja)](ja/README.md)
-* [Korean (ko)](ko/README.md)
-* [Lao (lo)](lo/README.md)
-* [Macedonian (mk)](mk/README.md)
-* [Malagasy (mg)](mg/README.md)
-* [Marathi (mr)](mr/README.md)
-* [Polish (pl)](pl/README.md)
-* [Portuguese (pt-BR)](pt-BR/README.md)
-* [Romanian (ro)](ro/README.md)
-* [Scottish Gaelic (gd)](gd/README.md)
-* [Serbian (sr)](sr/README.md)
-* [Sinhala (si)](si/README.md)
-* [Slovenian (sl)](sl/README.md)
-* [Spanish (es)](es/README.md)
-* [Spanish, Chile (es-CL)](es-CL/README.md)
-* [Spanish, Spain (es-ES)](es-ES/README.md)
-* [Swahili (sw)](sw/README.md)
-* [Tagalog (tl)](tl/README.md)
-* [Telugu (te)](te/README.md)
-* [Thai (th)](th/README.md)
-* [Turkish (tr)](tr/README.md)
-* [Ukrainian (uk)](uk/README.md)
-* [Vietnamese (vi)](vi/README.md)
-* [Wolof (wo)](wo/README.md)
-* [Xhosa (xh)](xh/README.md)
-* [Yoruba (yo)](yo/README.md)
-* [Zulu (zu)](zu/README.md)
+All localization style guides and related documents are listed in [SUMMARY.md](SUMMARY.md). If you’re reading the GitBook [version with improved readability](https://mozilla-l10n.github.io/styleguides), a menu with the list is on the left side of your screen.
```

Comment:
nit: is on the left side of your screen. -> is displayed on the left of the page.

Reply:
I have amended the commit with the change you suggested.